tests/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Makefile for running tests
+#
+
+SHELL:=bash
+.ONESHELL:
+.PHONY: env
+
+export LZMBRANCH:=$(shell git rev-parse --abbrev-ref HEAD)
+
+PREFIX:=lizmap-$(LZMBRANCH)-tests
+
+LIZMAP_USER_ID ?= $(shell id -u)
+LIZMAP_GROUP_ID ?= $(shell id -g)
+LZMPOSTGISVERSION ?= 11-2.5
+LZMQGSRVVERSION ?= 3.16
+LZMPGPORT ?= 8132
+LZMQGSRVPORT ?= 8131
+LZMWEBPORT ?= 8130
+LIZMAP_ADMIN_LOGIN ?= admin
+LIZMAP_ADMIN_EMAIL ?= admin@localhost.local
+LIZMAP_ADMIN_DEFAULT_PASSWORD_SOURCE ?= /srv/etc/admin.conf
+
+env:
+	@echo "Creating environment file for docker-compose"
+	@cat <<-EOF > .env
+	LIZMAP_USER_ID=$(LIZMAP_USER_ID)
+	LIZMAP_GROUP_ID=$(LIZMAP_GROUP_ID)
+	LZMPOSTGISVERSION=$(LZMPOSTGISVERSION)
+	LZMQGSRVVERSION=$(LZMQGSRVVERSION)
+	LZMBRANCH=$(LZMBRANCH)
+	LZMPGPORT=$(LZMPGPORT)
+	LZMQGSRVPORT=$(LZMQGSRVPORT)
+	LZMWEBPORT=$(LZMWEBPORT)
+	LIZMAP_ADMIN_LOGIN=$(LIZMAP_ADMIN_LOGIN)
+	LIZMAP_ADMIN_EMAIL=$(LIZMAP_ADMIN_EMAIL)
+	LIZMAP_ADMIN_DEFAULT_PASSWORD_SOURCE=$(LIZMAP_ADMIN_DEFAULT_PASSWORD_SOURCE)
+	COMPOSE_PROJECT_NAME=$(PREFIX)
+	EOF
+
+
+build: 
+	if [ ! -d qgis-server-plugins/lizmap ]; then
+		./add_server_plugins.sh
+	fi
```

Comment:
`docker-compose build`, is missing, isn't it?

Reply:
no, this is aimed to reproduce the  `run-test` command. building the images is done elsewhere and the docker-compose  does not handle build.

Reply:
The original script `./run-docker build` did the build by running `docker-compose build` with all environment variables.

Reply:
nop, see the diff.

Reply:
https://github.com/3liz/lizmap-web-client/pull/2573/files#diff-1da304854c4b19a64a89f66263f40f1060ea91eb6f7a0656483b80f192e9bcbbR16

Reply:
@dmarteau when I `./run-docker stop`, containers are recreated and I can see them as started with `docker ps -a`.